packages/components/src/VirtualizedList/ListTable/ListTable.component.js

```diff
@@ -86,19 +84,18 @@ ListTable.propTypes = {
 	id: PropTypes.string,
 	isActive: PropTypes.func,
 	isSelected: PropTypes.func,
+	noRowsRenderer: PropTypes.func,
```

Comment:
Should it be `noRowsRenderer: PropTypes.oneOf(PropTypes.element, PropTypes.func),` as it is in `PropTypes.js` ?

Reply:
No it is passed by RendererSelector that inject a function no matter what (see your next comment)